### ISSUE TITLE
Why some glyphs are displayed incorrectlty if we use Roboto via  google link? Is it possible to use all glyphs in the font via link?

### ISSUE THREAD

**Issue by SampetovaN** [googlefonts/roboto#339]
For example, how infinity symbol looks like if we add file .ttf (or woff(2), whatever) in a project 
![file-font](https://user-images.githubusercontent.com/38349890/127765563-f31ccba4-990e-48a7-816f-f9f00f1f8564.png)
But it doesn't work with the link
![font-link](https://user-images.githubusercontent.com/38349890/127765645-e17b879f-69e5-4722-8a16-7e1f8a9afdc1.png)

Why? Or it should work like this?

**Comment by josephrocca** [comment]
Yeah I just stumbled across this problem too. I'm pretty sure it's just falling back to another font that you have on your system - which is why it's a different glyph than you expect. Otherwise you'd see "tofu" - i.e. a square block indicating a missing glyph.

The worse part is that they don't actually tell you on the Google Fonts page that you're only getting a partial version of Roboto. Confused me for a while. You can get the "full" versions of Roboto here: https://github.com/googlefonts/roboto/tree/main/src/hinted

I don't think it's currently possible to get the full version via Google Fonts.
